sdks/models/src/main/avro/occurrence-hdfs-record.avsc

```diff
@@ -734,6 +734,26 @@
       "type" : "array",
       "items" : [ "string", "null" ]
     }, "null" ]
+  }, {
+    "name" : "datasettitle",
+    "type" : [ "string", "null" ]
+  }, {
+    "name" : "parenteventgbifid",
+    "type" : [ {
+      "type" : "array",
+      "items" : [ {
+        "type" : "record",
+        "name" : "ParentEventGbifId",
```

Comment:
shouldn't this be in lower case too? or no need?

Reply:
this schema is generated from the occurrence project, I'll look to see how easy is change that to provide better names for record types